plugins/vlc-video/vlc-video-plugin.c

```diff
@@ -24,8 +24,10 @@ LIBVLC_MEDIA_NEW_PATH libvlc_media_new_path_;
 LIBVLC_MEDIA_NEW_LOCATION libvlc_media_new_location_;
 LIBVLC_MEDIA_ADD_OPTION libvlc_media_add_option_;
 LIBVLC_MEDIA_RELEASE libvlc_media_release_;
-LIBVLC_MEDIA_RELEASE libvlc_media_retain_;
+LIBVLC_MEDIA_RETAIN libvlc_media_retain_;
```

Comment:
this just seems to have been a copy/paste error

Reply:
Is this change related to the other changes in this PR?

Reply:
Only in passing. There's no functional difference because they both resolve to the same type

```
typedef void (*LIBVLC_MEDIA_RETAIN)(libvlc_media_t *p_md);
typedef void (*LIBVLC_MEDIA_RELEASE)(libvlc_media_t *p_md);
```

Note that vlc-video-plugin.h already declares it as

```
extern LIBVLC_MEDIA_RETAIN libvlc_media_retain_;
```

So it's pretty clearly just a typo. Happy to revert if preferred.

Reply:
Revert isn't needed. I was just trying to understand it better to decide if it warranted its own commit so that `git blame` is clearer.